contrib/mesos/pkg/executor/config/config.go

```diff
@@ -22,8 +22,9 @@ import (
 
 // default values to use when constructing mesos ExecutorInfo messages
 const (
-	DefaultInfoID         = "k8sm-executor"
-	DefaultInfoSource     = "kubernetes"
-	DefaultInfoName       = "Kubelet-Executor"
-	DefaultSuicideTimeout = 20 * time.Minute
+	DefaultInfoID            = "k8sm-executor"
+	DefaultInfoSource        = "kubernetes"
+	DefaultInfoName          = "Kubelet-Executor"
+	DefaultSuicideTimeout    = 20 * time.Minute
+	DefaultLaunchGracePeriod = 5 * time.Minute
```

Comment:
clarification: when flag defaults are assigned. See how DefaultSuicideTimeout is used (NewKubeletExecutorServer)